migration-roadmap/status-light.md

```diff
@@ -0,0 +1,259 @@
+# Status Light migration roadmap
+
+## Component specifications
+
+### CSS
+
+<details>
+<summary>CSS selectors</summary>
+
+- `.spectrum-StatusLight`
+- `.spectrum-StatusLight--accent:before`
+- `.spectrum-StatusLight--blue:before`
+- `.spectrum-StatusLight--celery:before`
+- `.spectrum-StatusLight--chartreuse:before`
+- `.spectrum-StatusLight--cyan:before`
+- `.spectrum-StatusLight--fuchsia:before`
+- `.spectrum-StatusLight--gray:before`
+- `.spectrum-StatusLight--green:before`
+- `.spectrum-StatusLight--indigo:before`
+- `.spectrum-StatusLight--info:before`
+- `.spectrum-StatusLight--magenta:before`
+- `.spectrum-StatusLight--negative:before`
+- `.spectrum-StatusLight--neutral`
+- `.spectrum-StatusLight--neutral:before`
+- `.spectrum-StatusLight--notice:before`
+- `.spectrum-StatusLight--orange:before`
+- `.spectrum-StatusLight--positive:before`
+- `.spectrum-StatusLight--purple:before`
+- `.spectrum-StatusLight--red:before`
+- `.spectrum-StatusLight--seafoam:before`
+- `.spectrum-StatusLight--sizeL`
+- `.spectrum-StatusLight--sizeM`
+- `.spectrum-StatusLight--sizeS`
+- `.spectrum-StatusLight--sizeXL`
+- `.spectrum-StatusLight--yellow:before`
+- `.spectrum-StatusLight:before`
+- `.spectrum-StatusLight:lang(ja)`
+- `.spectrum-StatusLight:lang(ko)`
+- `.spectrum-StatusLight:lang(zh)`
+
+</details>
+
+<details>
+<summary>Passthroughs</summary>
+
+None found for this component.
+
+</details>
+
+<details>
+<summary>Modifiers</summary>
+
+- `--mod-statuslight-border-width`
+- `--mod-statuslight-content-color-default`
+- `--mod-statuslight-corner-radius`
+- `--mod-statuslight-dot-size`
+- `--mod-statuslight-font-size`
```

Comment:
We're missing mods for: font-family, font-style, brown, cinnamon, pink, silver, turquoise colors. 

and then the mod for `semantic-accent-color` should be removed.